Samples/NetworkMediaStreamer/RTSPServer/inc/SocketWrapper.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+#pragma once
+
+constexpr LPWSTR g_lpPackageName = (LPWSTR)UNISP_NAME;
+
+#define SEC_SUCCESS(Status) ((Status) >= 0)
+
+class CSocketWrapper

```

Comment:
>CSocketWrapper [](start = 6, length = 14)

final?

Reply:
i want people to be able to extend and add functionality. e.g. using a different API for secure or socket communication etc.

---
In reply to: [491109190](https://github.com/microsoft/Windows-Camera/pull/13#discussion_r491109190) [](ancestors = 491109190)